src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationUnsuccessfulPostGateway.cshtml

```diff
@@ -0,0 +1,223 @@
+﻿@using SFA.DAS.ApplyService.Domain.Entities
+@using SFA.DAS.ApplyService.Application.Apply.Roatp
+@using SFA.DAS.ApplyService.Domain.Apply
+@model SFA.DAS.ApplyService.Web.ViewModels.Roatp.ApplicationSummaryWithModeratorDetailsViewModel
+
+@{
+    ViewBag.Title = "Application unsuccessful";
+    Layout = "_Layout";
+
+    var passFailStatusCssClass = "status--pass";
+
+    @if (Model.FinancialReviewStatus == FinancialReviewStatus.Fail)
+    {
+        passFailStatusCssClass = "status--fail";
+    }
+}
+
+<partial name="~/Views/Roatp/ApplicationHeaderReadOnly.cshtml" />
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <div class="govuk-panel app-panel--error">
+                <h1 class="govuk-panel__title">
+                    Application unsuccessful
+                </h1>
+            </div>
+
+            <partial name="~/Views/Roatp/_ApplicationDetailsTable.cshtml" model="@Model" />
+
+            <p class="govuk-body">
+                Your application to join the Register for apprenticeship training providers (RoATP) has been unsuccessful.
+            </p>
+
+            <p class="govuk-body">
+                Your application has been reviewed by our team of assessors. You must pass every question for your application to be successful.
+            </p>
+            <p class="govuk-body">
+            You may wish to <a href="https://www.gov.uk/guidance/roatp-application-guidance-digital-may-2021" target="_blank">refer to the application guidance</a> to identify opportunities where your organisation may be able to improve on its policies, processes or approach.
+            </p>
+            <h1 class="govuk-heading-l">Feedback on your financial health assessment</h1>
+            <p class="govuk-body">
+                The outcome of your financial health assessment is:<br />
+                <span class="@passFailStatusCssClass hint--no-color">@Model.FinancialGrade</span>
```

Comment:
I'd probably have in-lined this or moved the code above down to here as it took me a few seconds to see how this was being calculated